pages/api/calendar.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon';
+
+import { getTalks } from '../../lib/airtable';
+
+const currentDate = DateTime.local();
+
+const removeOldTalks = talk => new Date(talk.date) > new Date();
+
+const futureTalks = month => ({
```

Comment:
Same here, maybe something more like `createTalk` or `createTalkOnMonth` ? Also, it's creating a talk for a set month, but not for the last wednesday of the month

Reply:
> Also, it's creating a talk for a set month, but not for the last wednesday of the month
We don't care about the wednesday because we only display mon name. And idk how to it with luxon, I can't find magic function (date-fns have it :D)

Reply:
> > Also, it's creating a talk for a set month, but not for the last wednesday of the month
We don't care about the wednesday because we only display mon name. And idk how to it with luxon, I can't find magic function (date-fns have it :D)

Reply:
> > > Also, it's creating a talk for a set month, but not for the last wednesday of the month

We don't care about the wednesday because we only display mon name. And idk how to it with luxon, I can't find magic function (date-fns have it :D)

Reply:
But a talk has a full date, so the mock up talk should have a full date too, especially since it's made on the API side, while the usage of it (And only of its month) is on the client side
Down the line, if we use it elsewhere/some other way, it might prove counter intuitive to have a "malformed" date

Reply:
I will do this in another PR, and I remove luxon. It's so hard to do what I want with it :/